dashboards/aws-apprunner/aws-apprunner.json

```diff
@@ -0,0 +1,301 @@
+          "visualization": {
+            "id": "viz.area"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "SELECT average(`aws.apprunner.MemoryUtilization`) FROM Metric WHERE `collector.name` = 'cloudwatch-metric-streams' FACET `aws.accountId` SINCE 24 HOURS AGO TIMESERIES"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            }
+          }
+        },
+        {
+          "title": "Application logs ",
+          "layout": {
+            "column": 1,
+            "row": 7,
+            "width": 4,
+            "height": 3
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.line"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "SELECT count(*) FROM Log WHERE `aws.logGroup` LIKE '%/application' SINCE 1648578495571 UNTIL 1648664890571 facet `aws.logGroup` TIMESERIES MAX"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            },
+            "yAxisLeft": {
+              "zero": true
+            }
+          }
+        },
+        {
+          "title": "Application logs",
+          "layout": {
+            "column": 5,
+            "row": 7,
+            "width": 8,
+            "height": 3
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "logger.log-table-widget"
+          },
+          "rawConfiguration": {
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "SELECT message FROM Log WHERE aws.logGroup LIKE '/aws/apprunner/%' AND aws.logGroup LIKE '%/application'"
+              }
+            ]
+          }
+        },
+        {
+          "title": "Service logs ",
+          "layout": {
+            "column": 1,
+            "row": 10,
+            "width": 4,
+            "height": 3
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.line"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "SELECT count(*) FROM Log WHERE `aws.logGroup` LIKE '%/service' SINCE 1648578245099 UNTIL 1648664640099 FACET `aws.logGroup` TIMESERIES MAX"
```

Comment:
```suggestion
                "query": "SELECT count(*) FROM Log WHERE `aws.logGroup` LIKE '%/service' SINCE 30 minutes ago FACET `aws.logGroup` TIMESERIES MAX"
```

Reply:
same issue here.